internal/backend/remote-state/s3/backend.go

```diff
@@ -563,14 +575,6 @@ func (b *Backend) Configure(obj cty.Value) tfdiags.Diagnostics {
 		cfg.SharedCredentialsFiles = []string{val}
 	}
 
-	if val, ok := boolAttrOk(obj, "skip_metadata_api_check"); ok {
```

Comment:
I'm a bit lost why this is gone, should we mention that in the changelog?

Reply:
Forgot to add a note. It was a duplicate. The same block exists a couple lines earlier.